item-spec/common-metadata.md

```diff
@@ -0,0 +1,136 @@
+| start_datetime | string | The first or start date and time for the item, in UTC. It is formatted as `date-time` according to [RFC 3339, section 5.6](https://tools.ietf.org/html/rfc3339#section-5.6). |
+| end_datetime   | string | The last or end date and time for the item, in UTC. It is formatted as `date-time` according to [RFC 3339, section 5.6](https://tools.ietf.org/html/rfc3339#section-5.6). |
+
+
+## Licensing
+
+Information about the license(s) of the data, which not the necessarily the same license that applies to the metadata.
+**Licensing information should be defined at the Collection level if possible.**
+
+- [JSON Schema](json-schema/license.json)
+
+| Field Name | Type   | Description |
+| ---------- | ------ | ----------- |
+| license    | string | Item's license(s), either a SPDX [License identifier](https://spdx.org/licenses/), `various` if multiple licenses apply or `proprietary` for all other cases. Should be defined at the Collection level if possible. |
+
+**license**: Data license(s) as a SPDX [License identifier](https://spdx.org/licenses/). Alternatively, use
+`proprietary` (see below) if the license is not on the SPDX license list or `various` if multiple licenses apply.
+In all cases links to the license texts SHOULD be added, see the [`license` link relation type](#relation-types).
+If no link to a license is included and the `license` field is set to `proprietary`, the collection is private,
+and consumers have not been granted any explicit right to use the data.
+
+#### Relation types
+
+| Type         | Description                                                  |
+| ------------ | ------------------------------------------------------------ |
+| license      | The license URL(s) for the item SHOULD be specified if the `license` field is set to `proprietary` or `various`. If there is no public license URL available, it is RECOMMENDED to supplement the STAC Item with the license text in a separate file and link to this file. |
+
+
+## Provider
+
+Information about the organizations capturing, producing, processing, hosting and publishing this data.
+**Provider information should be defined at the Collection level if possible.**
+
+- [JSON Schema](json-schema/provider.json)
+
+| Field Name | Type   | Description |
+| ---------- | ------ | ----------- |
+| providers  | [[Provider Object](#provider-object)] | A list of providers, which may include all organizations capturing or processing the data or the hosting provider. Providers should be listed in chronological order with the most recent provider being the last element of the list.  |
+
+#### Provider Object
+
+The object provides information about a provider. A provider is any of the organizations that captured or processed the content of the collection and therefore influenced the data offered by this collection. May also include information about the final storage provider hosting the data.
```

Comment:
"A provider is any of the organizations that captured or processed the content of the collection and therefore influenced the data offered by this collection", change it to present tense to match the rest of the document: "A provider is any of the organizations that captures or processes"